src/Appwrite/Utopia/Response/Model/Session.php

```diff
@@ -160,6 +160,12 @@ public function __construct()
                 'default' => false,
                 'example' => true,
             ])
+            ->addRule('secret', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Secret used to authenticate the user. Only included if the request was made with an API key',
+                'default' => '',
+                'example' => '5e5bb8c16897e',
+            ])
```

Comment:
We're going to need to be very careful and not leak this 😬

Reply:
We could do it differently:
How about: 
1) adding a new rule option in Model object called `admin` (or similiar) that defaults to false
2) In the response->dynamic() function we could filter out fields with admin=true, unless the user is privileged

Reply:
Maybe...I'm not sure what the best approach is. Maybe we need to make sure we have tests to catch it? 🤷🏼‍♂️